package.json

```diff
@@ -3,27 +3,27 @@
   "version": "0.1.0",
   "private": true,
```

Comment:
In the future please do not include packages update as part of another PR
It should be a dedicated PR